test/extensions/filters/network/thrift_proxy/integration_test.cc

```diff
@@ -46,12 +46,7 @@ class ThriftConnManagerIntegrationTest
             route_config:
               name: "routes"
               routes:
-                - match:
-                    method_name: "execute"
-                  route:
-                    cluster: "cluster_0"
-                - match:
-                    method_name: "poke"
+                - match: {}
```

Comment:
Oops. Could you revert the original and switch method_name to method? That should eliminate the unknown fields. (I'll circle back and add a second cluster to this test and use it for the one of the routes.)

Reply:
Actually, that'll just make half the tests fail. You can leave this and I'll fix it myself.

Reply:
Yes, I am getting errors if I put it back...